### PR TITLE
Add Task.using and CanClose.

### PR DESCRIPTION
**Pull request by leifwickland**

### CODE REVIEW

File: src/main/tut/README.md [hunk 1]
```diff
@@ -146,6 +146,16 @@ logging.
   .attemptRun)
 ```
 
+#### using
+Ensure that a resource is "closed" when a task completes, regardless of whether it's successful.
```
Comment (skeet70) [inline src/main/tut/README.md:150]: Might want to mention here that it requires a CanClose trait, and that Closeable and ExecutorService have predefined ones.

**Review by skeet70 — CHANGES_REQUESTED** [review]

File: src/main/tut/README.md (hunk 1 above)
Comment (leifwickland) [reply src/main/tut/README.md:150]: Good point. Words added.

**Review by skeet70 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -252,4 +268,4 @@ For those with permission to release:
     * Mac users can `brew install gpg pinentry-mac` to get the tools needed.
 * Create a Sonatype [credentials file](http://www.scala-sbt.org/1.0/docs/Using-Sonatype.html#Fourth+-+Adding+credentials).
 * Run `sbt release`
-* Push the newly created tags and version bump commits to `rubicon-project/rubiz`. 
+* `git push` to add the new tags and release commits to master.
```
Comment (coltfred) [inline README.md:271]: Looks like this got screwed up again.
@skeet70 Did you change this in the README for tut? Maybe Leif should change it on this branch too?

**Review by coltfred — APPROVED** [review]
LGTM.

**Pull request merged**